### PR TITLE
Troubleshooting guides update

### PR DESCRIPTION
**Pull request by mmitoraj**
**Description**

Changes proposed in this pull request:

- Rephrased some parts of the troubleshooting guides.

### CODE REVIEW

File: docs/04-operation-guides/troubleshooting/apix-02-dns-mgt-connection-refused.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: External DNS management - connection refused
 
 ## Symptom
 
-If you use a custom domain, you receive the `connection refused` error when you try to expose a service. It shows up when you call the service endpoint by sending a GET request. The error looks as follows:
+After all the steps required to prepare your custom domain are finished, you receive the `connection refused` error when you try to expose a service. It shows up when you call the service endpoint by sending a GET request. The error looks as follows:
```
Comment (Tomasz-Smelcerz-SAP) [inline docs/04-operation-guides/troubleshooting/apix-02-dns-mgt-connection-refused.md:7]: In the "Remedy" section (I can't comment there) I would add:

In addion, ensure that your OS resolves the target host name to the same Ingress Gateway IP address.
run:
```
   host httpbin,kyma-goat.ga
```

File: docs/04-operation-guides/troubleshooting/apix-03-dns-mgt-could-not-resolve-host.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ The error could result from:
 
 - Timing issues during the DNS Entry creation
 - VPN connection
-- Invalid DNS settings on your OS.
+- Invalid DNS settings on your OS
```
Comment (Tomasz-Smelcerz-SAP) [inline docs/04-operation-guides/troubleshooting/apix-03-dns-mgt-could-not-resolve-host.md:19]: In the "Remedy" section (I can't comment there):
- Add info that DNSEntry status should be: `Ready`
- Add a point: "Check if your local DNS configuration in /etc/hosts (or an equivalent file on your OS) contains an entry for the target host. Remove it."

File: docs/04-operation-guides/troubleshooting/apix-03-dns-mgt-could-not-resolve-host.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ The error could result from:
 
 - Timing issues during the DNS Entry creation
 - VPN connection
```
Comment (Tomasz-Smelcerz-SAP) [inline docs/04-operation-guides/troubleshooting/apix-03-dns-mgt-could-not-resolve-host.md:18]: - VPN connection (issues related to DNS servers managed by VPN provider)

File: docs/04-operation-guides/troubleshooting/apix-02-dns-mgt-connection-refused.md (hunk 1 above)
Comment (Tomasz-Smelcerz-SAP) [reply docs/04-operation-guides/troubleshooting/apix-02-dns-mgt-connection-refused.md:7]: `connection refused` or `connection timeout`

**Review by Tomasz-Smelcerz-SAP — DISMISSED** [review]

File: docs/04-operation-guides/troubleshooting/apix-02-dns-mgt-connection-refused.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
 ---
-title: External DNS management - connection refused
+title: External DNS management - connection refused or timeout
 ---
 
 ## Symptom
 
-If you use a custom domain, you receive the `connection refused` error when you try to expose a service. It shows up when you call the service endpoint by sending a GET request. The error looks as follows:
+After all the steps required to prepare your custom domain are finished, you receive the `connection refused` or `connection timeout` error when you try to expose a service. It shows up when you call the service endpoint by sending a GET request. The error looks as follows:
```
Comment (klaudiagrz) [inline docs/04-operation-guides/troubleshooting/apix-02-dns-mgt-connection-refused.md:7]: Please add the link to the tutorial on how to prepare your custom domain.

File: docs/04-operation-guides/troubleshooting/apix-03-dns-mgt-could-not-resolve-host.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ title: External DNS management - could not resolve host
 
 ## Symptom
 
-If you use a custom domain, you receive the `could not resolve host` error when you try to expose a service. It shows up when you call the service endpoint by sending a GET request. The error looks as follows:
+After all the steps required to prepare your custom domain are finished, you receive the `could not resolve host` error when you try to expose a service. It shows up when you call the service endpoint by sending a GET request. The error looks as follows:
```
Comment (klaudiagrz) [inline docs/04-operation-guides/troubleshooting/apix-03-dns-mgt-could-not-resolve-host.md:7]: Please add the link to the tutorial on how to prepare your custom domain.

File: docs/04-operation-guides/troubleshooting/apix-02-dns-mgt-connection-refused.md [hunk 6]
```diff
@@ -21,3 +21,10 @@ Check if the IP address provided as the value of the **spec.targets** parameter
 ```bash
 kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}'`
 ```
+
+In addion, ensure that your OS resolves the target host name to the same Ingress Gateway IP address.
+Run:
+
+```bash
+host httpbin.kyma-goat.ga
```
Comment (klaudiagrz) [inline docs/04-operation-guides/troubleshooting/apix-02-dns-mgt-connection-refused.md:29]: As discussed, please make it more generic.

File: docs/04-operation-guides/troubleshooting/apix-04-dns-mgt-resource-ignored.md [hunk 7]
```diff
@@ -4,7 +4,7 @@ title: External DNS management - resource ignored by the controller
 
 ## Symptom
 
-If you use a custom domain to expose a service and you have a non-Gardener clutser, the DNSProvider or DNSEntry CR you cretaed are ignored by the controller.
+If you have a non-Gardener cluster, the DNSProvider or DNSEntry CR you created are ignored by the controller.
```
Comment (klaudiagrz) [inline docs/04-operation-guides/troubleshooting/apix-04-dns-mgt-resource-ignored.md:7]:
```suggestion
If you have a non-Gardener cluster, the DNS Provider or DNS Entry CR you created are ignored by the controller.
```

Comment (klaudiagrz) [reply docs/04-operation-guides/troubleshooting/apix-04-dns-mgt-resource-ignored.md:7]: ^ You write it with space in other docs. Whatever option you decide on, please make it consistent in all the docs.

**Review by klaudiagrz — APPROVED** [review]

**Review by klaudiagrz — CHANGES_REQUESTED** [review]
Please revert the last commit

**Review by klaudiagrz — APPROVED** [review]

**Pull request merged**